src/geometry/manifold/ManifoldGeometry.h

```diff
@@ -16,7 +19,7 @@ class ManifoldGeometry : public Geometry
   VISITABLE_GEOMETRY();
 
   ManifoldGeometry();
-  ManifoldGeometry(const std::shared_ptr<const manifold::Manifold>& object);
+  ManifoldGeometry(const std::shared_ptr<const manifold::Manifold>& object, const std::map<uint32_t, Color4f> & originalIDToColor = std::map<uint32_t, Color4f>());
```

Comment:
Nit: you could probably just do `= {}` instead of assigning a fully specified type